### PR TITLE
adding redstone_dust sprites

### PR DESCRIPTION
**Pull request by WilliamHarmonMS**
Brilliant redstone dust sprites can now be used to make redstone dust blocks in levels.

### CODE REVIEW

File: src/js/game/LevelMVC/LevelView.js [hunk 1]
```diff
@@ -1139,6 +1141,7 @@ export default class LevelView {
           }
         }
 
+        //pelican
```
Comment (joshlory) [inline src/js/game/LevelMVC/LevelView.js]: Extra comment.

File: src/js/game/LevelMVC/LevelBlock.js [hunk 2]
```diff
@@ -93,6 +93,14 @@ export default class LevelBlock {
       this.isDestroyable = false;
       this.isTransparent = true;
     }
+    
+    if (blockType === "redstone_dust") {
+      this.isEntity = true;
+      this.isWalkable = true;
+      this.isUsable = true;
+      this.isDestroyable = true;
```
Comment (joshlory) [inline src/js/game/LevelMVC/LevelBlock.js]: Destroyable?

File: src/js/game/LevelMVC/LevelView.js [hunk 3]
```diff
@@ -2001,6 +2004,22 @@ export default class LevelView {
...
+        sprite = plane.create(xOffset + 40 * x, yOffset + plane.yOffset + 40 * y, atlas, frame);
+        frameList = Phaser.Animation.generateFrameNames("TNTexplosion", 0, 8, "", 0);
+        sprite.animations.add("explode", frameList, 7, false).onComplete.add(() => {
+          this.playExplosionCloudAnimation([x, y]);
+          sprite.kill();
+          this.toDestroy.push(sprite);
+          this.actionPlaneBlocks[this.coordinatesToIndex([x, y])] = null;
+        });
+        break;
+
```
Comment (joshlory) [inline src/js/game/LevelMVC/LevelView.js]: I don't think you need this section at all.  The `default:` case on line 2023 should account for the added block type.

**Comment by joshlory** [comment]
Screenshot for context: (placeholder assets)

image

File: src/js/game/LevelMVC/LevelBlock.js (hunk 2 above)
Comment (WilliamHarmonMS) [inline src/js/game/LevelMVC/LevelBlock.js]: The student will be able to place and destroy redstone dust (redstone -wire- I've been told it should be called). So I figured that was the right flag?

File: src/js/game/LevelMVC/LevelView.js (hunk 3 above)
Comment (WilliamHarmonMS) [inline src/js/game/LevelMVC/LevelView.js]: I was a bit unclear on this one. There are some unneeded things there (like the TNTexplosion reference) but before I added that, the createBlock was failing on redstone_dust.

It does seem like it should work without it, so I'll remove it and make sure things are still behaving properly. Perhaps there was some other issue that was unrelated.

**Pull request merged**